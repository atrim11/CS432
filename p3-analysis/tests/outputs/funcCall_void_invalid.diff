--- outputs/funcCall_void_invalid.txt	2024-10-25 13:42:23.052636467 -0400
+++ expected/funcCall_void_invalid.txt	2024-10-25 13:35:59.802297189 -0400
@@ -0,0 +1 @@
+Type mismatch: int expected but void found on line 5
