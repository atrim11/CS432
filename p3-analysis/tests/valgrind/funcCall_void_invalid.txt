==816616== Memcheck, a memory error detector
==816616== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==816616== Using Valgrind-3.22.0 and LibVEX; rerun with -h for copyright info
==816616== Command: ../decaf inputs/funcCall_void_invalid.decaf
==816616== 
Type mismatch: int expected but void found on line 5
==816616== 
==816616== HEAP SUMMARY:
==816616==     in use at exit: 0 bytes in 0 blocks
==816616==   total heap usage: 1,923 allocs, 1,923 frees, 230,184 bytes allocated
==816616== 
==816616== All heap blocks were freed -- no leaks are possible
==816616== 
==816616== For lists of detected and suppressed errors, rerun with: -s
==816616== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
