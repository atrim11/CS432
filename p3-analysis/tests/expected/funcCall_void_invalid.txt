Type mismatch: int expected but void found on line 5
